docs/otel/export-metrics.md

```diff
@@ -0,0 +1,64 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: OTel Export
+--->
+
+# Semantic Conventions for OTel Export Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+This document describes instruments and attributes for OTel
+Export level metrics. Consider the [general metric semantic
+conventions](README.md#general-metric-semantic-conventions) when creating
+instruments not explicitly defined in the specification.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Metric Instruments](#metric-instruments)
+  * [Metric: `otel.processor.spans`](#metric-otelprocessorspans)
+  * [Metric: `otel.exporter.spans`](#metric-otelexporterspans)
+
+<!-- tocstop -->
+
+## Metric Instruments
+
+### Metric: `otel.processor.spans`
+
+This metric is [required][MetricRequired].
+
+<!-- semconv metric.otel.processor.spans(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `otel.processor.spans` | Counter | `{span}` | Measures the number of processed Spans. |
+<!-- endsemconv -->
+
+<!-- semconv metric.otel.processor.spans(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `processor.dropped` | boolean | Whether the Span was dropped or not. [1] |  | Required |
```

Comment:
Is this missing the `otel.` prefix? So `otel.processor.dropped` instead of `processor.dropped`?